frontend/src/components/Forms/DetailsForm/index.js

```diff
@@ -14,7 +14,7 @@ const DetailsForm = ({ error, handleChange, name, email }) => {
     >
       <FormFieldLayout>
         <FormField
-          label={<FormLabel id="name">Name</FormLabel>}
+          label={<FormLabel id="name">Name*</FormLabel>}
```

Comment:
Shall we do this in a different PR? Would also be nice to render these based on the `required` attribute. Google a bit around how those are styles in React/CSS.


https://www.w3schools.com/tags/att_input_required.asp

Reply:
This would also require talking to Hannah. So no sure if we actually want to do this?